openedx/core/djangoapps/content_tagging/rest_api/v1/tests/test_objecttag_export_helpers.py

```diff
@@ -0,0 +1,189 @@
+"""
+Test the objecttag_export_helpers module
+"""
+from unittest.mock import patch
+
+from openedx_tagging.core.tagging.models import Tag
+from organizations.models import Organization
+
+from common.djangoapps.student.tests.factories import UserFactory
+from xmodule.modulestore.tests.django_utils import TEST_DATA_SPLIT_MODULESTORE, ModuleStoreTestCase
+
+from .... import api
+from ..objecttag_export_helpers import TaggedContent, build_object_tree_with_objecttags, iterate_with_level
+
+
+class TaggedCourseMixin(ModuleStoreTestCase):
+    """
+    Mixin with a course structure and taxonomies
+    """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    def setUp(self):
```

Comment:
It looks like this setUp is very similar to [TestContentObjectChildrenExportView.setUp](https://github.com/openedx/edx-platform/pull/34091/files#diff-297a8accac446967f48cca923cbf48ebf1f37f1b5d34235edc01d7922c588349R1637)?

Could these classes share a mixin instead of duplicating code?

Reply:
We are using the shared `TaggedCourseMixin` in both.

Maybe you are talking about `TestGetAllObjectTags`?
https://github.com/openedx/edx-platform/blob/309ce943808ea57b7c43db88020728629bf07264/openedx/core/djangoapps/content_tagging/tests/test_api.py#L253-L257

We created a similar structure on this one but didn't need the store (only the Keys). I didn't like it either, but I also didn't wanna bring the store mixin overhead to the API tests. Do you have any suggestion?

Reply:
Sure, see https://github.com/open-craft/edx-platform/pull/623